### ISSUE TITLE
[UI] Public Docker Image and Helm Chart #51

### ISSUE THREAD

**Issue by dennyglee** [unitycatalog/unitycatalog#433]
**Is your feature request related to a problem? Please describe.**
As part of the migration of the unitycatalog-ui repo to unitycatalog (main), referring the original UI issue: [Public Docker Image and Helm Chart #51](https://github.com/unitycatalog/unitycatalog-ui/issues/51)

**Describe the solution you would like**

**Additional context**

**Comment by dennyglee** [comment]
FYI @almogtavor, @jaceklaskowski , @itsLuized

**Comment by jaceklaskowski** [comment]
I can work on the docker part (which I've been thinking of anyway once I noticed this UI code drop).

**Comment by JuanPabloDiaz** [comment]
You can write the word "take", and it will be auto-assigned to you @jaceklaskowski

**Comment by jaceklaskowski** [comment]
Yeah @JuanPabloDiaz, you're 💯  correct! The only reason I haven't done it yet is because I didn't know if the others mentioned would like to take on it (and perhaps don't know that it's just `take` away 👍 )

**Comment by JuanPabloDiaz** [comment]
> Yeah @JuanPabloDiaz, you're 💯 correct! The only reason I haven't done it yet is because I didn't know if the others mentioned would like to take on it (and perhaps don't know that it's just `take` away 👍 )

Gotcha 👍

**Comment by dennyglee** [comment]
Go for it @jaceklaskowski :)

**Comment by sdwbgn** [comment]
I've cooked up a prototype of Helm chart for Unity Catalog (Server + UI) - https://github.com/sdwbgn/unitycatalog-helm
Currently it's using manually built images from fresh `main` branch, but when docker images issue will be done, I would be happy to bring it to main UC repository.
For now feel free to check it out - any feedback is welcome!

**Comment by dennyglee** [comment]
This is super exciting @sdwbgn !  Thanks to @tnightengale , we're working on the docker images (currently blocked on me).  Working on this as soon as I can, eh?!

**Comment by hongbo-miao** [comment]
We are also waiting the Docker image and Helm chart, thanks!

**Comment by dennyglee** [comment]
We're almost finished with the docker image in docker hub per #936

**Comment by hongbo-miao** [comment]
Hi @dennyglee hmm, today the Unity Catalog has Docker image, but Unity Catalog UI is still missing Docker image.

**Comment by unfrgivn** [comment]
> Hi [@dennyglee](https://github.com/dennyglee) hmm, today
> 
> * Unity Catalog
>   
>   * Docker image: ready
>   * Helm chart: [in progress](https://github.com/unitycatalog/unitycatalog/pull/895)
> * Unity Catalog UI
>   
>   * Docker image: missing
>   * Helm chart: missing
> 
> Note the Unity Catalog UI is still missing Docker image. Thanks!

I'm happy to open a PR with my production Docker UI build if no one is working on this

**Comment by unfrgivn** [comment]
@hongbo-miao I'll take the heart as a "yes"? The contributing docs said I should ask first before opening a PR. My only issue as I got into it is I have completely removed the UI proxy configuration on my local version so I would need to add that back in a way that allows for backwards compatibility. There is some related discussion in #899

**Comment by hongbo-miao** [comment]
Hi @unfrgivn, yeah, that would be awesome if you could help! 😃 I think if we need to let anyone know in advance, it should probably be @dennyglee.
For us, Unity Catalog was published first, but now it is falling behind.
It is making us start to think about switching to Iceberg and the Polaris catalog, since Polaris used to be behind but is now ahead, offering both a Docker image and a Helm chart.
